### PR TITLE
Add Variable benchmarks

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            services.AddSingleton<MyQueryGraphType>();
+            services.AddSingleton<MyInputObjectGraphType>();
+            services.AddSingleton<MySubInputObjectGraphType>();
+            services.AddSingleton<ISchema, MySchema>();
+
+            _provider = services.BuildServiceProvider();
+            _schema = _provider.GetRequiredService<ISchema>();
+            _schema.Initialize();
+            _executer = new DocumentExecuter();
+            _queryLiteral = @"
```
Comment (sungam3r) [inline src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs:42]: Let's keep all such requests in separate files like here:
изображение

I moved them there to make it easier to edit them separately from the code. In addition, then syntax highlighting becomes available. Also no need in escaping like `""`.

Comment (sungam3r) [reply src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs:42]: https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql

File: src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+            Variable();
+            EnableValidation = val;
+        }
+
+        public IEnumerable<bool> TrueFalse => new[] { true, false };
+
+        [ParamsSource(nameof(TrueFalse))]
+        public bool EnableValidation { get; set; }
+
+        [Benchmark]
```
Comment (sungam3r) [inline src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs:150]:
```suggestion
        [Benchmark(Baseline = true)]
```

**Comment by sungam3r** [comment]
👍 👍 👍

File: src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs (hunk 1 above)
Comment (Shane32) [reply src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs]: That's a great idea.  For now, I just consolidated the queries in the same fashion as is done in the rest of this project.

File: src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs (hunk 2 above)
Comment (Shane32) [reply src/GraphQL.Benchmarks/Benchmarks/VariableBenchmark.cs:61]: Committed, then reverted.  Produces meaningless output.

**Review by sungam3r — APPROVED** [review]

**Pull request merged**